_posts/en/newsletters/2020-11-25-newsletter.md

```diff
@@ -74,7 +74,29 @@ development branch and so those changes will likely not be released
 until version 0.22, about six months after the release of the upcoming
 version 0.21.*
 
-- [Bitcoin Core #20305][] and [Bitcoin Core #20426][] wallet: introduce fee_rate sat/vB param/option FIXME:jonatack
+- [Bitcoin Core #20305][] begins standardizing RPC fee rate units to satoshis
+  per vbyte (sat/vB), marking a shift away from BTC/kvB, by introducing an
+  `fee_rate` param/option denominated in sat/vB to the `sendtoaddress`,
+  `sendmany`, `fundrawtransaction` and `walletcreatefundedpsbt` RPCs as well as
+  to the experimental new `send` RPC. In addition, the `bumpfee` RPC `fee_rate`
+  option is changed from BTC/kB to sat/vB. Users are warned that the latter is a
+  breaking API change, but it should be relatively benign: the large (10,000
+  times) difference between BTC/kvB and sat/vB units means that a transaction
+  with a fee rate mistakenly set in sat/vB rather than BTC/kvB should generally
```

Comment:
This sounds a bit weird to me, since the parameter key determines the unit. Perhaps, it would be better to phrase it as "if a transaction is accidentally created with a fee rate expressed in `BTC/kvB` when the RPC expects `sat/vB`".

Reply:
Thanks for the catch! I wrote it backward.